src/glass/src/app/i18n.helper.ts

```diff
@@ -35,8 +36,10 @@ export class TranslateHttpLoader implements TranslateLoader {
 
 export const supportedLanguages: Record<string, string> = {
   /* eslint-disable @typescript-eslint/naming-convention */
-  en_US: 'English',
-  de_DE: 'Deutsch'
+  en_US: TEXT('English'),
```

Comment:
I'm not sure if we should translate them. Imagine you accidentally switch to Chinese language  and then `Deutsch` is displayed in Chinese letters. Each language should be hard-coded:
```
en_US => English
de_DE => Deutsch
zh_CN (Chinese) => 中國人
zh_TW (Taiwanese) => 台灣
```
@AvengerMoJo Please correct my Google translations.

Reply:
According to @AvengerMoJo you should use
```
zh_CN (Simplified) => 简体中文
zh_TW (Traditional) => 繁體中文
```